.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Your PR title should start with one of the following conventions:
```

Comment:
"Your PR title must follow [conventional commits](), and should start with one of the following [types]():"